content/1_General/Generic_Code.mdx

```diff
@@ -34,20 +34,18 @@ competitor.
 		title="Making a Contest Template"
 	/>
 	<Resource
-		source="Benq"
+		source="BenQ"
```

Comment:
why?

Reply:
i mean q is your last name, first & last should start with a capital?
i can change it back tho

Reply:
personally I never capitalize the last letter of my username :)

Reply:
alr then